docs/CONTRIBUTING.MD

```diff
@@ -1,11 +1,19 @@
-## Pastel Electron Wallet
+### Pastel Wallet Fullnode - Contributing Guidelines
 
-To start for development, run:
+Pastel Wallet Fullnode is an Electron app built with React on the frontend.
 
-```
-yarn start
-```
+##### Getting started
 
-### Important
+You should have Node LTS and Yarn installed globally. After project clone, run `yarn` to install the dependencies. To run the daemon service `pasteld`, you can select the proper binary in `static/bin`, or just download and run the existing official Pastel Wallet Fullnode app from website. Finally run `yarn start` to start the Electron app locally in development mode.
```

Comment:
You don't need to start `pasteld` manually, the app will launch it. Here is where it happens: https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/components/LoadingScreen.tsx#L341

Reply:
This links gives me a 404